src/convertToObject.js

```diff
@@ -3,16 +3,44 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+  const mass = [];
+  const arrSplitFirstLevel = sourceString.split(';');
+
+  for (const words of arrSplitFirstLevel) {
+    const arrSplitSecondLevel = words.split(':');
+
+    for (const values of arrSplitSecondLevel) {
+      const cut = values.trim();
+
+      mass.push(cut);
+    }
+  }
+
+  const filtered = mass.filter(function(elem) {
+    if (elem !== '') {
+      return true;
+    } else {
+      return false;
+    }
+  });
```

Comment:
```suggestion
  const filtered = mass.filter(Boolean);
```

`''` empty string will be transformed to `false`, because `Boolean('') === false` and vice versa